lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -41,19 +42,16 @@
 #endif
 
 namespace {
-	UsdPrimSiblingRange filteredChildren( const UsdPrim& prim )
+	UsdPrimSiblingRange getUSDFilteredChildren(const UsdPrim& prim, const Usd_PrimFlagsPredicate pred = UsdPrimDefaultPredicate)
```

Comment:
Added the predicate as input (with default) so that it can be used by new filtered children method.